packages/js-widgets-compiler/src/moveWidgetDependencies.js

```diff
@@ -20,6 +20,12 @@ module.exports = async (widget, origin, destination) => {
   debug(
     `[info] - [${shortcode}] Installing dependencies including devDependencies.`,
   );
+  await runCommand('nvm use', [], {
+    cwd: origin,
+    scope: shortcode,
+    successMessage:
+      "Checking for .nvmrc file. Switching to widget's configured Node version if found.",
+  });
```

Comment:
Can we wrap this with a try ... catch?

In this case we want to ignore any errors derived from the command execution. The main case being that `nvm` is not installed.

Reply:
@e0ipso Yes, indeed. Updated.

Incidentally, I'm not sure this will actually work. Might not be able to call `nvm` from within a child process.

Couple of links with info:

- https://github.com/nvm-sh/nvm/issues/540
- https://stackoverflow.com/questions/46762937/nodejs-how-to-change-node-version-with-nvm-while-using-child-processs-spawn

Perhaps you can advise on the proper method for accomplishing the Node version check and switch.

Reply:
> Incidentally, I'm not sure this will actually work. Might not be able to call nvm from within a child process.

Now that you mention it, we may have already tried this in the past without success. :thinking:

Sadly, I don't think we can solve this. This will not be a problem once we transition to `@js-widgets/webpack-cli`.